test/Core.Test/Services/OrganizationServiceTests.cs

```diff
@@ -138,5 +146,216 @@ public async Task OrgImportCreateNewUsersAndMarryExistingUser()
             await orgUserRepo.Received(1).UpsertAsync(Arg.Any<OrganizationUser>());
             await orgUserRepo.Received(2).CreateAsync(Arg.Any<OrganizationUser>());
         }
+
+        [Theory, CustomAutoDataAttribute(typeof(SutProviderCustomization))]
```

Comment:
not important, but you can remove the work `Attribute` from these references (keep Attribute in the class name)

Reply:
Done